### PR TITLE
Chat bubbles

### PR DESCRIPTION
**Pull request by Agrosis**

### CODE REVIEW

**Comment by niksudan** [comment]
HYPE

File: src/app/components/chat/bubbles/bubbles-directive.js [hunk 1]
```diff
@@ -41,21 +41,9 @@ angular.module( 'App.Chat' ).directive( 'gjChatTabs', function( Screen, Chat, Sh
 				Chat.client.leaveRoom( roomId );
 			};
 
-			$scope.$watch( 'Chat.client.hasOpenRooms() && !Screen.isXs', function( isShowing )
+			$scope.$watch( '!Screen.isXs', function( isShowing )
```
Comment (hworld) [inline src/app/components/chat/bubbles/bubbles-directive.js]: This is silly. In stylus just put a breakpoint in the styles that effectively do what chat-tabs-visible does?

```stylus
display: none
@media $media-sm-up
  display: block
```

File: src/app/components/chat/bubbles/bubbles.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div ng-if="ctrl.isShowing">
```
Comment (hworld) [inline src/app/components/chat/bubbles/bubbles.html]: Screen is on the scope, so you can do `ng-if="!Screen.isXs"`

**Comment by hworld** [comment]
Needs translations. Ask me how to do this when you get online.

**Pull request merged**